packages/office-ui-fabric-react/src/components/Link/Link.types.ts

```diff
@@ -20,4 +27,22 @@ export interface ILinkProps extends React.AllHTMLAttributes<HTMLAnchorElement |
    */
   disabled?: boolean;
 
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  getStyles?: IStyleFunction<ILinkProps, ILinkStyles>;
+
+  /**
+   * Theme (provided through customization.)
+   */
+  theme?: ITheme;
+}
+
+export interface ILinkStyleProps {
+  disabled?: boolean;
+  theme: ITheme;
 }
+
+export interface ILinkStyles {
+  root?: IStyle;
```

Comment:
everything in IFooStyles should be required, not optional

Reply:
I've updated this and documented in our [merge-styles guidelines](https://github.com/OfficeDev/office-ui-fabric-react/wiki/SCSS-to-mergeStyles).